i18n/zh-CN.toml

```diff
@@ -27,36 +23,36 @@ all_tags = "所有标签"
 # Post metadata.
 draft = "草稿"
 min_read = "分钟阅读"
-words = "单词"
+words = "字"
 last_updated_on = "最后更新于"
-see_changes = "查看更改"
+see_changes = "修改纪录"
 
 # Post body.
 table_of_contents = "目录"
-load_comments = "加载评论"
+load_comments = "载入留言"
 
 # Footer: Powered by Zola and tabi.
-powered_by = "由"
+powered_by = "网站基于"
 and = "和"
-site_source = "网站源代码"
+site_source = "查看原始码"
 
 # 404 error.
 # https://welpo.github.io/tabi/404.html
 page_missing = "您请求的页面似乎不存在"
 translation_missing = "或尚未翻译成您的语言"
-check_url = "请检查URL是否有误或"
+check_url = "请检查网址是否有误或"
 go_home = "返回首页"
 
 # For multilingual quote shortcode.
 # https://welpo.github.io/tabi/blog/shortcodes/#multilingual-quotes
-show_original_quote = "显示原文引用"
+show_original_quote = "显示原文"
 show_translation = "显示译文"
-open_quotation_mark = "“"
-close_quotation_mark = "”"
+open_quotation_mark = "「"
+close_quotation_mark = "」"
 
 # Translations for stylised Atom feed.
 # https://welpo.github.io/tabi/atom.xml
 # Must contain "About Feeds"; it will become a link.
-about_feeds = "这是一个网页订阅源，也称为Atom订阅源。通过复制地址栏中的URL到您的新闻阅读器订阅。访问 About Feeds 了解更多并开始使用。免费。"
-visit_the_site = "访问网站"
+about_feeds = "这是Web Feed，又称为Atom Feed，把现在的网址复制到新闻阅读器即可订阅本站文章。造访「About Feed」来了解更多资讯。"
```

Comment:
should contain "About Feeds" (missing the s in "Feeds") so it becomes a link in the [Stylized feed](https://welpo.github.io/tabi/atom.xml)